src/client/helpers/entity.tsx

```diff
@@ -227,10 +227,9 @@ export function authorCreditToString(authorCredit) {
 }
 
 export function getEntityDisambiguation(entity) {
-  if (entity.disambiguation) {
-	if(entity.disambiguation.comment){
+	if(entity.disambiguation?.comment){
 		return <small>{` (${entity.disambiguation.comment})`}</small>;
-	}else{
+	} else if(entity.disambiguation){
 		return <small>{` (${entity.disambiguation})`} </small>;
 	}
   }
```

Comment:
There is now a superfluous closing curly bracket } that will make this file fail.